x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-overview.json

```diff
@@ -374,7 +374,7 @@
                         }
                     }
                 },
-                "title": "AWS EC2 Instance State ECS",
+                "title": "EC2 Instance State [Metricbeat AWS] ECS",
```

Comment:
Do you mind remove the ECS part here please? Or I can create a separate PR for it 😬

Reply:
I am explictly keeping it because it was adding during the ECS migration, it is needed to help differentiate ECS from non-ECS dashboards.
Or this dashboard didn't exist before ECS?

Reply:
yeah this dashboard didnt exist before ECS so no need to add ECS here. I can create a separate PR to fix this 😬 so it won't confuse other people who review this PR.